driver.cpp

```diff
@@ -1,6 +1,32 @@
+#include <iostream>
 #include "log.cpp"
 #include "description.cpp"
+#include "validation.cpp"
 
-int main(){
+using namespace std;
+
+void printLogFileData() {
+	for (auto &log : Logs::logs) {
+		cout << "File: " << log.name << endl;
+		cout << "ClassID: " << log.classId << endl;
+		cout <<  "First Name: " << log.firstName << endl;
+		cout <<  "Last Name: " << log.lastName << endl;
+
+		for (auto &activity : log.activities) {
+			cout << "\tCode: " << activity.code << ", ";
+			cout << "Date: " << activity.date << ", ";;
+			cout << "Duration: " << activity.minutes << endl;
+		}
+	}
+}
```

Comment:
We can remove this later, but it's useful to demonstrate how to access the log file data once validation is passed.